### DISCUSSION TITLE
Duplicating starred repositories in feed

### DISCUSSION THREAD

**Discussion by LWJerri** [community/community#69080] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/50290430/7e5f3617-c3c2-4373-9f19-667677c885b3)

**Comment by Satont** [comment]
![telegram-cloud-photo-size-2-5251538568625638656-y](https://github.com/community/community/assets/42675886/debf59d1-0a8d-46f5-baa5-91d2af6fd196)

**Comment by whonion** [comment]
![image](https://github.com/community/community/assets/47848969/c6a43553-2eb0-41cb-bbab-f47eebe19d60)
